backend/app/categories/controller.py

```diff
@@ -33,8 +48,27 @@ async def get_category_item_route(category_id: str, item_id: ObjectID,
 async def add_item_route(category_id: str, item: Union[Movie, Anime, Show, Music],
                          db: AsyncIOMotorClient = Depends(get_database)) -> ItemInResponse:
     """Add item to a given category"""
-    # TODO add item to db
-    return {"success": True}
+    if category_id not in CategoryEnum.__members__:
```

Comment:
Not required with the latest commit, please pull upstream

Reply:
Removed now

Reply:
It still hasn't been removed!

Reply:
Sorry I thought i removed it